release/Magarena/scripts/Aetherstorm_Roc.groovy

```diff
@@ -1,37 +1,47 @@
 def action = {
     final MagicGame game, final MagicEvent event ->
-    game.doAction(new ChangeCountersAction(event.getPermanent(), MagicCounterType.PlusOne, 1));
     if (event.isYes()) {
-        event.processTargetPermanent(game, {
-            game.doAction(new TapAction(it));
-        });
+        game.doAction(new ChangeCountersAction(event.getPlayer(), MagicCounterType.Energy, -2));
+        game.doAction(new ChangeCountersAction(event.getPermanent(), MagicCounterType.PlusOne, 1));
+        game.doAction(new TapAction(event.getRefPermanent()));
     }
 }
 
 [
     new ThisAttacksTrigger() {
         @Override
         public MagicEvent executeTrigger(final MagicGame game,final MagicPermanent permanent, final MagicPermanent attacker) {
-            return (permanent.getController().getCounters(MagicCounterType.Energy) >= 2) ?
-                new MagicEvent(
-                    permanent,
-                    new MagicMayChoice("Pay {E}{E}?"),
-                    this,
-                    "PN may\$ pay {E}{E}."
-                ):
-                MagicEvent.NONE;
+            return new MagicEvent(
+                permanent,
+                new MagicMayChoice("Target a creature to tap?", TARGET_CREATURE_YOUR_OPPONENT_CONTROLS),
```

Comment:
The prompt should mention defending player, perhaps "Choose target creature defending player controls?"